app/models/checkout.rb

```diff
@@ -30,16 +34,28 @@ def recalled?
     recalled_date.present?
   end
 
-  def patron_key
-    fields['patron']['key']
+  def claims_returned_date
+    Time.zone.parse(fields['claimsReturnedDate']) if fields['claimsReturnedDate']
+  end
+
+  def claims_returned?
+    claims_returned_date.present?
+  end
+
+  def renewal_date
+    Time.zone.parse(fields['renewalDate']) if fields['renewalDate']
   end
```

Comment:
Again, another potential `private` method since we don't display this. Private methods generally don't need to be tested either, or can be tested indirectly (by the method invoking it) which is why I bring it up mainly.

Reply:
for this PR, renewal date actually is not required, I will remove it for this, and include it with renewals.

Reply:
Oh yeah, you're right. 👍